src/hooks/data.tsx

```diff
@@ -31,11 +31,11 @@ export type Data = {
     weekday: boolean;
   };
   indexes: {
-    distance: Index[];
-    region: Index[];
-    time: Index[];
-    type: Index[];
-    weekday: Index[];
+    distance: any[];
+    region: any[];
+    time: any[];
+    type: any[];
+    weekday: any[];
```

Comment:
do we need this typescript change?

Reply:
No we don't!, I temporary changed it and forgot to put back. thanks